x/evm/genesis.go

```diff
@@ -18,16 +18,12 @@ func InitGenesis(
 	ctx sdk.Context,
```

Comment:
reminder to self: read genesis.json specification since it represents the state of all the modules when imported/exported